core/app/models/spree/payment/processing.rb

```diff
@@ -206,7 +206,7 @@ def handle_response(response)
       end
 
       def record_response(response)
-        log_entries.create!(details: response.to_yaml)
+        log_entries.create!(parsed_details: response)
```

Comment:
I had to double-check that `create!` will accept the `parsed_details` attribute even if it's not a table field. Non-blocking, but what do you think if we make the implementation and interface clearer by creating another method like `Spree::LogEntry.log` or similar?